bundle/metadata/dependencies.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- type: olm.package
+  value:
+    packageName: ramendr
+    version: ">4.10.0"
```

Comment:
@ShyamsundarR Is this package information correct?

Reply:
Should be >= as the version that ships would be 4.10.0?